Automattic/newspack-blocks
# Set up the blocks to be translated, and add a POT file

### All Submissions:


### Changes proposed in this Pull Request:

This PR starts the process of translating the Newspack Blocks.

In it I:

* Added the text-domain `newspack-blocks`, to the JS files (the text-domain that the blocks orginially used, `newspack`, didn't work for the JS, so I:
* Updated the text domain in the .php files from `newspack` to `newspack-blocks`
* Unfortunately included formatting changes that came from my Prettier plugin; if this makes it a pain to review, I can redo the changes on fresh files without the formatting changes.

See #74 

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build:webpack`
2. Confirm that the text-domain changes throughout the JS and PHP are correct.
3. Make sure the POT file appears to include strings from JS and PHP files.

### Other information:
